tools/pinning/current/pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Certbot Project"]
 license = "Apache License 2.0"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.7"
 
```

Comment:
The `traitlets` pinning in this file could be dropped I think? Running repin to check ...